lib/logflare_web/live/search_live/log_event_components.ex

```diff
@@ -0,0 +1,74 @@
+defmodule LogflareWeb.SearchLive.LogEventComponents do
+  use LogflareWeb, :html
+  use LogflareWeb, :routes
+
+  use Phoenix.Component
+
+  import LogflareWeb.ModalLiveHelpers
+  import Logflare.Utils, only: [iso_timestamp: 1]
+  import LogflareWeb.Helpers.BqSchema
+
+  alias Logflare.DateTimeUtils
+  alias Phoenix.LiveView.JS
+
+  @log_levels ~W(debug info warning error alert critical notice emergency)
+
+  attr :log, :map
+  attr :search_timezone, :string
+  attr :query_string, :string
+  attr :source, :map
+
+  def log_search_event(%{log: %{body: body}} = assigns) when is_map_key(body, "event_message") do
+    tz_part =
```

Comment:
do we still need this given we don't show the tz part any more in the timestamp? can potentially eliminate this.

Reply:
The timestamp is still shown in the content for copy to clipboard:

```heex
        <.link class="tw-text-[0.65rem]  group-hover:tw-visible tw-invisible" phx-click={JS.dispatch("logflare:copy-to-clipboard", detail: %{text: "#{@formatted_timestamp}    #{@message}"})} data-toggle="tooltip" data-placement="top" title="Copy to clipboard">
          copy
        </.link>
```

https://github.com/Logflare/logflare/pull/2546/files#diff-3805bb32c63be339e8f8f7bf33182f7ab3c2719c8e724ce968d3bb96081de869R51-R53

If we don't need the timestamp there then, yes, it can be eliminated.

Reply:
ah ok lets keep it then